tools/SourceKit/tools/sourcekitd/lib/API/Requests.cpp

```diff
@@ -1834,6 +1834,8 @@ static void reportCursorInfo(const RequestResult<CursorInfoData> &Result,
       Parent.set(KeyUSR, ParentTy.USR);
     }
   }
+  if (Info.IsDynamic)
```

Comment:
I think an important piece of this is also providing the USR of the receiver type. This can be used to determine at which point in the class hierarchy we should start looking for overrides.

Reply:
Added a "receivers" since protocol composition could make this multiple types (even though we currently don't handle that case).